daprdocs/content/en/java-sdk-docs/_index.md

```diff
@@ -0,0 +1,230 @@
+
+try (DaprClient client = (new DaprClientBuilder()).build()) {
+  client.publishEvent(PUBSUB_NAME, TOPIC_NAME, message, singletonMap(Metadata.TTL_IN_SECONDS, MESSAGE_TTL_IN_SECONDS)).block();
+}
+```
+
+##### Subscribe to messages
+
+```java
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.dapr.Topic;
+import io.dapr.client.domain.CloudEvent;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+import reactor.core.publisher.Mono;
+
+@RestController
+public class SubscriberController {
+
+  private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
+
+  @Topic(name = "testingtopic", pubsubName = "${myAppProperty:messagebus}")
+  @PostMapping(path = "/testingtopic")
+  public Mono<Void> handleMessage(@RequestBody(required = false) CloudEvent cloudEvent) {
+    return Mono.fromRunnable(() -> {
+      try {
+        System.out.println("Subscriber got: " + cloudEvent.getData());
+        System.out.println("Subscriber got: " + OBJECT_MAPPER.writeValueAsString(cloudEvent));
+      } catch (Exception e) {
+        throw new RuntimeException(e);
+      }
+    });
+  }
+
+}
+```
+
+- For a full list of state operations visit [How-To: Publish & subscribe]({{< ref howto-publish-subscribe.md >}}).
+- Visit [Java SDK examples](https://github.com/dapr/java-sdk/tree/master/examples/src/main/java/io/dapr/examples/pubsub/http) for code samples and instructions to try out pub/sub
+
+### Interact with output bindings
+
+```java
+import io.dapr.client.DaprClient;
+import io.dapr.client.DaprClientBuilder;
+
+try (DaprClient client = (new DaprClientBuilder()).build()) {
+  // sending a class with message; BINDING_OPERATION="create"
+  client.invokeBinding(BINDING_NAME, BINDING_OPERATION, myClass).block();
+
+  // sending a plain string
+  client.invokeBinding(BINDING_NAME, BINDING_OPERATION, message).block();
+}
+```
+
+- For a full guide on output bindings visit [How-To: Use bindings]({{< ref howto-bindings.md >}}).
+- Visit [Java SDK examples](https://github.com/dapr/java-sdk/tree/master/examples/src/main/java/io/dapr/examples/bindings/http) for code samples and instructions to try out output bindings
+
+### Retrieve secrets
+
+```java
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.dapr.client.DaprClient;
+import io.dapr.client.DaprClientBuilder;
+import java.util.Map;
+
+try (DaprClient client = (new DaprClientBuilder()).build()) {
+  Map<String, String> secret = client.getSecret(SECRET_STORE_NAME, secretKey).block();
+  System.out.println(JSON_SERIALIZER.writeValueAsString(secret));
+}
+```
+
+- For a full guide on secrets visit [How-To: Retrieve secrets]({{< ref howto-secrets.md >}}).
+- Visit [Java SDK examples](https://github.com/dapr/java-sdk/tree/master/examples/src/main/java/io/dapr/examples/secrets) for code samples and instructions to try out retrieving secrets
+
+### Actors
```

Comment:
An example for Actor is beyond just the Actor class. Please, add a description about what Virtual Actors are and provide more links to existing docs explaining Actors. Then, mention that this is an example of an interface of a Virtual Actor in Java.